sdk/keyvault/azure-keyvault-certificates/seed.txt

```diff
@@ -0,0 +1 @@
+iscai-msft
```

Comment:
We should add this alongside the code that uses it, in another PR.

Reply:
I thought I deleted it but turns out I had it in two places, deleting this one now.